nc-times/src/api.js

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios'
+
+export function getAllArticles(){
+    return axios.get('https://project-nc-news-adrian-sartini.onrender.com/api/articles')
+    .then((data)=>{
+        return  data.data.articles
```

Comment:
This line isn't very readable. I'd suggest either de-structuring {data} on line 5, or naming whatever has been returned more generically (something like 'response') - at the moment, it's impossible to tell what each of the objects is doing.